src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,36 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder was called");
+        if (userId == null) {
+            logger.error("Incoming data for userId is null");
```

Comment:
you log error but essentially ignoring the fact that it is null, I wouldn't complicate things with this null check